app/models/foreman_openscap/arf_report.rb

```diff
@@ -5,6 +5,7 @@ class ArfReport < ::Report
     include Taxonomix
     include OpenscapProxyExtensions
 
+    # attr_accessible :host_id, :reported_at, :status, :metrics
```

Comment:
Updating foreman/develop helped, could you just make a small change? s/openscap_proxy_id/openscap_proxy/ in here:
https://github.com/theforeman/foreman_openscap/blob/master/app/models/concerns/foreman_openscap/openscap_proxy_extensions.rb#L7